cmd/authenticator/main.go

```diff
@@ -74,3 +76,7 @@ func printErrorAndExit(errorMessage string) {
 	errLogger.Printf(errorMessage)
 	os.Exit(1)
 }
+
+func printVersion(){
+	infoLogger.Printf("Kubernetes Authenticator Client v%s starting up...", authenticator.FullVersionName)
```

Comment:
If `printVersion only has this one line, it seems fine just inlining it w/ `main()`.

Reply:
good point, added to main